### PR TITLE
Ty/long lat

### PR DESCRIPTION
**Pull request by TangoYankee**
---
name: Pull Request
about: standardize back end references to geo coordinate systems as 'long lat'
title: ''
labels: ''
reviewers: ''

---

## Checklist
- [x] Add description
- [x] Reference open issue pull request addresses
- [x] Pass functional tests
  - complete on the local machine with `make local.shell` `make test`
- [x] Request code review
  - Please allow **36 hours** from opening a pull request before merging a pull request- even if it has already recieved an approving review.
- [x] Address comments on code and resolve requested changes
- [x] Merge own code

## Description
Issue: #187 

*Brief description of solution*
- GeoJSON and PostGIS follow long lat convention
- Update generation of fake data to follow this convention
- Change order of coordinate throughout dustrak and models to reinforce the convention.

### CODE REVIEW

File: api/woeip/apps/air_quality/models.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class Sensor(models.Model):
         ("PM10", "PM10"),
         ("PM2.5", "PM2.5"),
         ("μg/m3", "μg/m3"),
-        ("latlon", "latlon"),
+        ("lonlat", "lonlat"),
```
Comment (TangoYankee) [inline api/woeip/apps/air_quality/models.py:45]: - Appears to be breaking the migrations and will need to remain labeled as 'latlon'

**Comment by theecrit** [comment]
@TangoYankee Are you still working on this, or do you need a review?

**Comment by TangoYankee** [comment]
> @TangoYankee Are you still working on this, or do you need a review?

~~PR #189 is a blocker. It contains a fix that would make the tests pass. I'm waiting for it to be merged into master and then I will integrate its fixing changes. After that, it'll be ready for review.~~

I'm updating it to the fixed master to get it ready for review.

**Review by nbilenko — APPROVED** [review]
Looks good, thanks @TangoYankee

**Review by jayqi — APPROVED** [review]
LGTM

**Pull request merged**